frontend/src/component/common/DialogFormTemplate/DialogFormTemplate.styles.tsx

```diff
@@ -33,11 +33,14 @@ export const StyledHeader = styled(Typography)({
     fontWeight: 'lighter',
 });
 
-export const ProjectNameContainer = styled('div')({
+export const NameContainer = styled('div')(({ theme }) => ({
     gridArea: 'project-name',
-});
+    display: 'flex',
+    flexFlow: 'column nowrap',
+    gap: theme.spacing(4),
+}));
```

Comment:
The gap only shows up when the naming pattern is there. Without it, it still looks the same.